src/mensajes/entities/mensaje.entity.spec.ts

```diff
@@ -1,7 +1,7 @@
-import { Mensaje } from './mensaje.entity';
+import { Message } from './mensaje.entity';
```

Comment:
nice! 
please update also the files names:

`./mensaje.entity` => `./message.entity`